core/java8actionloop/CHANGELOG.md

```diff
@@ -19,9 +19,10 @@
 
 # Java 8 OpenWhisk Runtime Container
 
-## 1.15.0 (next release)
+## 1.15.0
   - Include latest security fixes with every build.
   - adoptopenjdk/openjdk8-openj9:x86_64-ubuntu-jdk8u262-b10_openj9-0.21.0
```

Comment:
```suggestion
  - Adopt openjdk/openjdk8-openj9:x86_64-ubuntu-jdk8u262-b10_openj9-0.21.0
```

Reply:
Sigh.  This proposed change is wrong.  adoptopenjdk is the name of the docker repo. https://github.com/apache/openwhisk-runtime-java/blob/master/core/java8actionloop/Dockerfile#L32